scripts/build.mjs

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env zx
+import { $, path, os, fs, argv, fetch, cd, within, chalk } from 'zx'
+import { finished } from 'node:stream/promises'
+import archiver from 'archiver'
+
+if (argv.h || argv.help) {
+  console.log(`${argv._} [--target <name of target>] [--node-version <version>]`)
+  console.log()
+  console.log('Targets can be "linux", "linux-amd64", "linux-arm64", "darwin", "darwin-amd64", "darwin-arm64".')
+  console.log('If target does not indicate an architecture it will build all architectures for that platform.')
+  console.log('Target may be specified multiple times to build multiple targets.')
+  console.log()
+  console.log('Node version should be in the format "v#.#.#". For example: "v18.4.0')
+  console.log()
+  process.exit(0)
+}
+
+const targets = (Array.isArray(argv.target) ? argv.target : [argv.target])
+  .filter(v => !!v)
+  .map(v => v.toLowerCase())
+
+const nodeVersion = argv.nodeVersion || process.version
+const matrix = [
+  // [platform, arch, download extension, target extension, mirror]
+  ['linux', 'amd64', '.tar.xz', '', ''],
+  // TODO: build for amd64 musl
+  // Tried using the unofficial amd64-musl build, but when you try to run that build of `node`
+  // in alpine container, we get lots of errors like:
+  // Error relocating /tmp/caxa/applications/gluctl/linux-amd64-musl-1659414624544/0/node_modules/.bin/node: _ZSt4cerr: symbol not found
+  // ['linux', 'amd64-musl', '.tar.xz', '', 'https://unofficial-builds.nodejs.org/download/release'],
```

Comment:
I tried to get it working in an alpine container, but when it tries to invoke node, I just get a bunch of errors 🤷🏻‍♂️ Maybe it would work if I just pulled the node alpine docker image and just grabbed the version of node out of there idk.

Reply:
I think it's fine, we can iterate on supporting this in the future if possible.